string1.py

```diff
@@ -4,7 +4,7 @@
 """
 # Your name, plus anyone who helped you with this assignment.
 # Give credit where credit is due.
-__author__ = "???"
+__author__ = "Leanne Benson"
```

Comment:
Next time, use your GitHub username here.

Reply:
Sounds good! Github Username is LeanneBenson. Is that what is expected? There was really nothing said about that in the rubric or any information regarding expectations for author title.

Reply:
It's not a Kenzie requirement, it's a GItHub convention. :)